packages/jss/src/renderers/DomRenderer.js

```diff
@@ -263,7 +263,7 @@ function insertStyle(style: HTMLElement, options: PriorityOptions) {
     // https://stackoverflow.com/questions/41328728/force-casting-in-flow
     const insertionPointElement: HTMLElement = (insertionPoint: any)
     const {parentNode} = insertionPointElement
-    if (parentNode) parentNode.insertBefore(style, insertionPointElement.nextSibling)
+    if (parentNode) parentNode.insertBefore(style, prevNode)
```

Comment:
`prevNode` will always be `null` here.

Reply:
Indeed it will, as will it in the "getHead().insertBefore" line below. Perhaps using appendChild for both is more descriptive?